nilearn/tests/test_signal.py

```diff
@@ -356,6 +356,15 @@ def test_clean_confounds():
     assert_raises(TypeError, nisignal.clean, signals,
                   confounds=[None])
 
+    # Test without standardizing that constant parts of confounds are
+    # accounted for
+    np.testing.assert_almost_equal(nisignal.clean(np.ones((20, 2)),
```

Comment:
Nitpick: would it not be slightly better to check that all the values are close to 0, i.e. removing the `.mean()`?